processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -19,15 +19,16 @@
 package org.mapstruct.ap.internal.conversion;
 
```

Comment:
I think that enforced an empty line between the package decl and the legal header, but I'll check again.